homeassistant/components/freebox/router.py

```diff
@@ -113,7 +113,8 @@ async def update_sensors(self) -> None:
         # According to the doc `syst_datas["sensors"]` is temperature sensors in celsius degree.
         # Name and id of sensors may vary under Freebox devices.
         for sensor in syst_datas["sensors"]:
-            self.sensors_temperature[sensor["name"]] = sensor["value"]
+            if "value" in sensor:
+                self.sensors_temperature[sensor["name"]] = sensor["value"]
```

Comment:
Does the sensor have a name?
If it does, it might make more sense to set the value to `None` like this:
```python
self.sensors_temperature[sensor["name"]] = sensor.get("value")
```

Reply:
I was thinking the same, I'll check that